charts/nexus3/templates/configmap-properties.yaml

```
@@ -7,6 +7,12 @@ metadata:
     {{- include "nexus3.labels" . | nindent 4 }}
 data:
   nexus.properties: |
+    {{- if .Values.license.enabled }}
+    nexus.licenseFile=/nexus-data/etc/license/{{ .Values.license.key }}
```

```suggestion
    nexus.licenseFile=/nexus-data/etc/nexus.license
```
Sorry I missed this in the review. The `licence.key` is only used in the mapping as we're mounting it to this absolute path.